django_rest/django_rest/views.py

```diff
@@ -0,0 +1,98 @@
+
+    @swagger_auto_schema(
+        operation_description="Create Employee.",
+        request_body=openapi.Schema(
+            type=openapi.TYPE_OBJECT,
+            required=['username', 'password', 'first_name', 'last_name', 'birthdate'],
+            properties={
+                'username': openapi.Schema(type=openapi.TYPE_STRING),
+                'password': openapi.Schema(type=openapi.TYPE_STRING),
+                'first_name': openapi.Schema(type=openapi.TYPE_STRING),
+                'last_name': openapi.Schema(type=openapi.TYPE_STRING),
+                'birthdate': openapi.Schema(type=openapi.TYPE_STRING)
+            },
+        ),
+        security=[]
+    )
+    def post(self, request):
+        serializer = EmployeeSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            user, created = User.objects.get_or_create(username=serializer.data['username'])
+            if created:
+                user.password = make_password(serializer.data['password'])
+                user.first_name = serializer.data['first_name']
+                user.last_name = serializer.data['last_name']
+                user.save()
+                employee = Employee.objects.create(user=user, birthdate=serializer.data['birthdate']).save()
+                return Response({"success": "Employee '{}' created successfully".format(employee)}, status=200)
+            else:
+                return Response({"error": "This username: '{}' already used".format(serializer.data['username'])}, status=409)
+
+    @swagger_auto_schema(
+        operation_description="Update Employee.",
+        request_body=openapi.Schema(
+            type=openapi.TYPE_OBJECT,
+            required=['username', 'password', 'first_name', 'last_name', 'birthdate'],
+            properties={
+                'username': openapi.Schema(type=openapi.TYPE_STRING),
+                'password': openapi.Schema(type=openapi.TYPE_STRING),
+                'first_name': openapi.Schema(type=openapi.TYPE_STRING),
+                'last_name': openapi.Schema(type=openapi.TYPE_STRING),
+                'birthdate': openapi.Schema(type=openapi.TYPE_STRING)
+            },
+        ),
+        security=[]
+    )
+    def put(self, request):
+        serializer = EmployeeSerializer(data=request.data)
+        if serializer.is_valid(raise_exception=True):
+            user = get_object_or_404(User, username=serializer.data['username'])
+            if user:
+                employee = get_object_or_404(Employee, user=user)
+                employee.user.password = serializer.data['username']
+                employee.user.first_name = serializer.data['first_name']
+                employee.user.last_name = serializer.data['last_name']
+                employee.birthdate = serializer.data['birthdate']
+                employee.save()
+                return Response({"success": "Employee '{}' updated successfully".format(employee)})
+
```

Comment:
This functionality should be in the [next story](https://mahonya.atlassian.net/browse/PRA-12).